### ISSUE TITLE
Led color of keyboard (g512) reset when unplug and re-plug usb port

### ISSUE THREAD

**Issue by lutuantai95** [MatMoul/g810-led#286]
Hi g810-led team,

I have a question. 

Is there a way to set up when I unplug and re-plug the usb port of keyboard (G512), the led color won't reset to default?

This doesn't happen on windows 10. I used [Logitech Gaming Software](https://support.logi.com/hc/en-au/articles/360025298053) to change color of keyboard.

Thanks in advanced for your response!

**Comment by mfvianna** [comment]
Yes, there is.  Use the udev rules provided along with the software.  Just be aware that the default file permissions on the provided udev (666) rules are dangerous, one should use 660 instead to prevent any user to sniff the keyboard events and, for example capture passwords.

**Comment by lutuantai95** [comment]
@mfvianna 
Thanks for your response!
Could you show me how to set up udev (660) rules (By CLI or images)?
Thanks!
